### PR TITLE
Chore data migration

### PR DESCRIPTION
**Pull request by jamesthebrooks**
## Changes proposed in this pull request:
- Migrated data from jekyll site to here

### CODE REVIEW

File: _data/collections.yaml
Comment (apburnes) [inline _data/collections.yaml:23]: You can remove this `cloud.gov knowledge base` config. It is a relic of the Decap CMS that is no longer used in the cloud.gov site.

File: _data/assetPaths.json
Comment (sknep) [inline _data/assetPaths.json:1]: I'm curious -- do we gain anything by having assetPaths checked in? in another project, we're looking at adding it to the gitignore because its so noisy in diffs. If you agree, could you delete it and add it to the ignore?

File: _data/collections.yaml
Comment (sknep) [inline _data/collections.yaml:7]: what are these descriptions for? do you know? I'm wondering if this is only used in decap.

**Review by sknep — APPROVED** [review]
Once you address the comments, go ahead and merge :)

File: _data/assetPaths.json
Comment (jamesthebrooks) [reply _data/assetPaths.json:1]: I have also been curious about this...  Let me get it ignored after the others have been merged.  I'll make a note.  Do we even need _site at all?

File: _data/collections.yaml
Comment (jamesthebrooks) [reply _data/collections.yaml:7]: It might even be example data?  I honestly don't know and will defer to @apburnes and you.

File: _data/assetPaths.json
Comment (jamesthebrooks) [reply _data/assetPaths.json:1]: Per Slack discussion, removing "_site" from tracking and adding an issue here for the Pages template: https://github.com/cloud-gov/pages-uswds-11ty/issues/78

**Review by apburnes — APPROVED** [review]

**Pull request merged**
